script/check_format

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+# Format code with black.
+
+cd "$(dirname "$0")/.."
```

Comment:
I think that is not needed because we execute scripts from main folder

Reply:
We do it for every script, just in case people don't do that .